lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -1224,6 +1224,16 @@ void PatchInOutImportExport::visitReturnInst(ReturnInst &retInst) {
 
     useLayer = enableMultiView || useLayer;
 
+    if (nextStage == ShaderStageFragment) {
+      const auto &nextBuiltInUsage = m_pipelineState->getShaderResourceUsage(ShaderStageFragment)->builtInUsage.fs;
+      useLayer = nextBuiltInUsage.layer || nextBuiltInUsage.viewIndex || useLayer;
```

Comment:
I think you can do `useLayer |= nextBuiltInUsage.layer || nextBuiltInUsage.viewIndex`

Reply:
Also a few lines above in the existing code